Tasks/AzureRmWebAppDeployment/operations/TaskParameters.ts

```diff
@@ -54,11 +60,38 @@ export class TaskParametersUtility {
 
         return taskParameters;
     }
+
+    private static _initializeDefaultParametersForPublishProfile(taskParameters: TaskParameters): void {
+        taskParameters.connectedServiceName = null;
+        taskParameters.WebAppName = null;
+        taskParameters.PublishProfilePath = tl.getInput('PublishProfilePath', true);
+        taskParameters.PublishProfilePassword = tl.getInput('PublishProfilePassword', true);
+        taskParameters.isLinuxApp = false;
```

Comment:
why do you need to initialize parameters other than that of interest?

Reply:
I have added it just for the sake of initialization. Now removed it.